.github/scripts/clone_engine_repo.conf

```diff
@@ -1 +1 @@
-DEFAULT_BRANCH=BABEL_2_X_DEV__PG_14_X
+DEFAULT_BRANCH=jira-babel-3682
```

Comment:
Temp change to test with engine PR change. Will remove this after it's ok to merge in whole.